CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ While the web viewer supports a variety of codecs, the native viewer supports on
 Read more about our video supports (and its limits) [in our video docs](https://rerun.io/docs/reference/video).
 
 ### ⚠️ Breaking changes
+* 🗾 Blueprint files (.rbl) from previous Rerun versions will no longer load
```

Comment:
nit: tiny bit confusing for me as explicit loading will still work, it's just that auto-loading of auto-saved files won't work?

Reply:
+1, I think this is confusing. We're unusually stable in this release actually. It's just that we always refuse to load older blueprints automatically

Reply:
I didn't realize - thanks for the clarification!